docs/using.rst

```diff
@@ -44,17 +24,62 @@ above method instead.
 
 .. _prerequisites:
 
-Prerequisites
-=============
+Installation and Usage
+======================
+
+To install and run the client you just need to type:
+
+.. code-block:: shell
+
+   ./letsencrypt-auto
+
+.. warning:: Please do **not** use ``python setup.py install``.  That mode of
+             operation might corrupt your operating system and is **not supported**
+             by the Let's Encrypt team!
```

Comment:
My thinking was that we should remove all of the parts of the documentation that tell people to explicitly do anything with venv and pip, and then they won't end up issuing mistaken commands involving those tools.